service/src/main/java/uk/gov/hmcts/reform/fpl/model/MigratedChildren.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class MigratedChildren {
+    private final ChildParty party;
+
```

Comment:
leadChild?

Reply:
....I've been avoiding adding this, do you think it's definitely needed?

Reply:
Caught up with @geek-soft, leadChild is not needed.